pkgs/applications/misc/todoist-electron/default.nix

```diff
@@ -36,7 +37,7 @@ stdenv.mkDerivation rec {
   '';
 
   postFixup = ''
-    makeWrapper ${electron_21}/bin/electron $out/bin/${pname} \
+    makeWrapper ${electron_24}/bin/electron $out/bin/${pname} \
```

Comment:
```suggestion
    makeWrapper ${electron_24}/bin/electron $out/bin/todoist-electron \
```

Reply:
This already works and the variable was in place already, what's the need to replace this?

Reply:
using pname wherever possible is an anti pattern and increases code complexity while also sneaking in potential errors with finalAttrs.